src/Microsoft.Azure.WebJobs.Protocols/FunctionDescriptor.cs

```diff
@@ -35,5 +36,31 @@ public class FunctionDescriptor
         /// </summary>
         [JsonIgnore]
         internal MethodInfo Method { get; set; }
+
+        /// <summary>
+        /// Gets the <see cref="TriggerParameterDescriptor"/> for this function
+        /// </summary>
+        [JsonIgnore]
+        internal TriggerParameterDescriptor TriggerDescriptor { get; set; }
```

Comment:
The class is shared between dashboard and host. These properties are set and consumed by the host.